config/token.go

```diff
@@ -0,0 +1,65 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"github.com/dgrijalva/jwt-go"
+	"time"
+)
+
+// Structured version of Claims Section, as referenced at
+// https://tools.ietf.org/html/rfc7519#section-4.1
+// See examples for how to use this with your own claim types
+type TokenClaim struct {
+	User string `json:"username"`
+	jwt.StandardClaims
+}
+
+type TokenGenerated struct {
+	Token     string `json:"token"`
+	ExpiredAt string `json:"expired_at"`
```

Comment:
We wanna set a expiration time only @josejulio in the claim against the configuration file

Reply:
Not a blocker, but this field is duplicated, set once in  the token (encoded) and once in the answer of `GenerateToken` (verbose).

Reply:
If you see in the handlerRoot you can see that Token generated is the object returned to the user, in tokenclaim is the info to generate the token and the second one the output to the user where I inform him about when the token will be expired

Reply:
Yes, i see that, i only mean to say that the expiresAt is also self-contained in the token (the unix timestamp) and the consumer can use that too, we don't have to pull out every data from the claims.
Again, not a blocker, but would like to make that clear in case we want to add more data to the claims.